src/components/details-view/details-view.js

```diff
@@ -442,4 +443,4 @@ class DetailsView extends Component {
   }
 }
 
-export default withRouter(DetailsView);
+export default DetailsView;
```

Comment:
Are `history` and `location` props still being passed to the component when using real environment? Please check it because there may be an issue here

Reply:
Yes, `history` and `location` props are still being passed